FirebaseRemoteConfig/Sources/RCNFetch.m

```diff
@@ -486,13 +486,13 @@ - (NSString *)constructServerURL {
   NSString *serverURLStr = [[NSString alloc] initWithString:kServerURLDomain];
   serverURLStr = [serverURLStr stringByAppendingString:kServerURLVersion];
 
-  if (_options.projectID) {
+  if (_options.gcmSenderID) {
```

Comment:
Although `GCMSenderID` is required, this could be a breaking change causing developer's fetches to fail if they don't have the appropriate `GCMSenderID` in their `FIROptions`.

Also, this should be `GCMSenderID` in Objective-C (and likely isn't compiling right now).

Reply:
Thanks Ryan. I'll hold off on this PR for now then.

Reply:
I'll rework this PR such that the SDK defaults to `GCMSenderID`, but then uses `ProjectID` if the `GCMSenderID` doesnt exist. Is that acceptable?